packages/opentelemetry-core/test/context/HttpTraceContext.test.ts

```diff
@@ -201,6 +229,10 @@ describe('HttpTraceContext', () => {
           '00-ffffffffffffffffffffffffffffffff-ffffffff-01',
         invalidSpanId_tooLong:
           '00-ffffffffffffffffffffffffffffffff-ffffffffffffffff0000-01',
+        invalidFutureVersion:
+          'ff-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01',
+        invalidFutureFieldAfterFlag:
+          'cc-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01.bad-future-is-imminent-for-you-jk',
```

Comment:
Suggest you change this to something more neutral sounding. Have had issues in the past where english/western humor did not translate well to other cultures.

Reply:
Will do, forgot I did this. Thanks.